koku/api/report/queries.py

```diff
@@ -86,7 +87,17 @@ class ProviderMap(object):
                         'default_ordering': {'total': 'desc'},
                     },
                     'instance_type': {
+                        'aggregate': {
+                            'cost': Sum('unblended_cost'),
+                            'count': Sum('resource_count'),
+                            'value': Sum('usage_amount'),
+                        },
                         'aggregate_key': 'usage_amount',
+                        'annotations': {'cost': Sum('unblended_cost'),
+                                        # The summary table already already has counts
+                                        'count': Sum('resource_count'),
+                                        'total': Sum('usage_amount'),
+                                        'units': Max('unit')},
                         'count': 'resource_count',
```

Comment:
I spotted a couple of code paths that still reference it, so I left it. I'd like to avoid scope creep and say that we can do another pass with a separate PR to remove those remaining references.

Reply:
I was unable to find any that still did. That is the reason I asked. I commented about it here because you are are working with counts in this PR and it didn't feel like scope creep to me, just related code maintenance. That said we can tackle this in another PR.

Reply:
I'll open a new bug report to make sure we circle back and clean this up.